<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate spoken Cantonese to written Chinese</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="appid_label">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>29</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>appid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="remember_checkBox">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>56</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remember Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="secretkey_label">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>53</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>secretKey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openfile_pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>211</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="filepath_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>211</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="trans_progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logging_textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>291</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This is an application to translate spoken Cantonese text to written Chinese text for SRT files using Baidu translation API.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;進行中英翻譯前，建議先將中文文本修改成更通順的語句，可以提高翻譯質量。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logging_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yue2cht_radioButton">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>104</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>yue-cht</string>
    </property>
    <property name="text">
     <string>粵語-&gt;繁體</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="zh2en_radioButton">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>164</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>zh-en</string>
    </property>
    <property name="text">
     <string>中文-&gt;英文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>23</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secretkey_lineEdit">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>53</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="appid_lineEdit">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>30</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="yue2zh_radioButton">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>135</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>yue-zh</string>
    </property>
    <property name="text">
     <string>粵語-&gt;简体</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress_label">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>268</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
